### DISCUSSION TITLE
Cannot change the color for options in SingleSelect / MultiSelect columns

### DISCUSSION THREAD

**Discussion by AlfredoRamos** [nocodb/nocodb#821] [Features]
Hi :wave:

I've been testing NocoDB and I see it has a lot of potential.

I've been creating  some SingleSelect and MultiSelect columns, however the button that I believe is for changing the color, in fact just deletes the option.

Not sure if it's a bug or it hasn't been implemented yet, since I see it's under active development.

<details><summary>Screenshot</summary>

![Screenshot_20211128_151634](https://user-images.githubusercontent.com/4165935/143786983-85b475b4-47f2-4eaf-ab5f-d1d72475785c.png)

</details>

My setup (Docker Compose installation):
- NocoDB v0.83.5
- PostgreSQL 14
- Nginx Mainline

**Comment by o1lab** [comment]
This is not yet possible, I believe there is an issue open for this. We will be handling it soon.

**Reply by JmiXIII** [reply]
would be appreciated as well as color on conditions

**Comment by lhyphendixon** [comment]
I came here and contributed specifically for this. (And more broadly, the ability to be able to color rows and columns.)  Thank you guys so much for NocoDB.  It's truly a gift.
